controllers/auth/logout.js

```diff
@@ -0,0 +1,20 @@
+const { User } = require("../../models");
+const { RequestError } = require("../../errorHandlers");
+
+const logout = async (req, res) => {
+  const { _id } = req.user;
+
+  const user = await User.findById(_id);
+  if (!user) {
+    throw new RequestError(401, `Not authorized`);
+  }
+
+  await User.findByIdAndUpdate(_id, { token: null });
+  res.status(204).json({
+    status: "success",
+    code: 204,
+    message: "No content",
+  });
+};
```

Comment:
```suggestion
  res.sendStatus(204)
```
якщо статус код 204, то res body ігнорується

Reply:
прорустила цей момент, дякую)